tests/deployments/conftest.py

```diff
@@ -0,0 +1,24 @@
+import pytest
+import entrypoints
+from . import dummy_plugin
+
+
+f_target = 'fake_target'
+
+
+def custom_entrypoint_group_all(name):
```

Comment:
I think it might simplify the tests + docs if we actually just included this as a real plugin under https://github.com/mlflow/mlflow/tree/master/tests/resources/mlflow-test-plugin/mlflow_test_plugin & registered it as an entrypoint in https://github.com/mlflow/mlflow/blob/master/tests/resources/mlflow-test-plugin/setup.py. Then we wouldn't need to mock out entrypoint logic here & could reference the fake deployment plugin in the main plugin docs